models/topics.model.js

```diff
@@ -16,4 +16,16 @@ function findArticleById(articleId){
     })
 }
 
-module.exports = {findALlTopics, findArticleById}
\ No newline at end of file
+function findAllArticles(){
+    // const commentCount = db.query(`SELECT COUNT(body) AS comment_count FROM comments WHERE `)
```

Comment:
Dont forget to remove console.logs and comments before pushing your code up to github